src/vs/base/common/winjs.base.d.ts

```diff
@@ -134,5 +134,5 @@ export declare class PPromise<C, P> extends TPromise<C> {
 	public static join<C, P>(promises: PPromise<C, P>[]): PPromise<C, P[]>;
 	public static join<C, P>(promises: { [n: string]: PPromise<C, P> }): PPromise<{ [n: string]: C }, P>;
 	public static any<C, P>(promises: PPromise<C, P>[]): PPromise<{ key: string; value: PPromise<C, P>; }, P>;
-	public static wrapError<V>(error: any): TPromise<V>;
+	public static wrapError(error: Error): TPromise<never>;
```

Comment:
Actually true, wasn't aware that `never` is assignable everywhere. This might be our first usage of never ...